docs/userguides/projects.md

```diff
@@ -0,0 +1,45 @@
+# Developing Projects
+
+Use `ape` to create blockchain projects.A common project structure looks like this:
+
+```
+project         # The root project directory
+├── contracts/  # Project source files, such as .sol or .vy files
+├── tests/      # Project tests, ran using the 'ape test' command
+└── scripts/    # Project scripts, such as deploy scripts, ran using the 'ape run <name>' command
+```
+
+# Compiling Contracts
+
+The project manager object is a representation of your current project. Access it from the root `ape` namespace:
+
+```python
+from ape import project
+```
+
+Your `project` contains all the "relevant" files, such as sources files in the `contracts/` directory. The 
+`contracts/` directory is where compilers look for contracts to compile. File extensions found within the `contracts/` 
+directory determine which compiler plugin ape uses. Make sure to intall the compiler plugins you need if they are 
+missing.
```

Comment:
```suggestion
missing by adding them to your `ape-config.yaml`'s `plugin` section, or manually adding via the following:
```

Reply:
Also maybe deep link the `ape-config.yaml` references to the config documentation

Speaking of which, do we _have_ config documentation? lol

Reply:
There is no centralized config documentation (yet anyway). We have some stuff in the `ConfigManager` method-docs though. I think including it in this project-related guide makes a lot of sense too, at least the out-of-the-box configurations you can set.